classes/class-amp-admanager.php

```diff
@@ -239,4 +239,104 @@ public function load_amp_resources() {
 
 		}
 	}
+
+	/**
+	 * To get amp ad html code for all breakpoints.
+	 *
+	 * @param array   $attr Attributes.
+	 * @param boolean $echo Whether to echo or return html code.
+	 *
+	 * @return string
+	 */
+	public static function get_ads( $attr = [], $echo = false ) {
+
+		if ( empty( $attr ) ) {
+			return;
+		}
+
+		$ad_html     = '';
+		$breakpoints = self::get_breakpoints( $attr['sizes'] );
+
+		foreach ( $breakpoints as $device => $breakpoint ) {
+
+			if ( empty( $breakpoint['sizes'] ) ) {
+				continue;
+			}
+
+			$attr['width']  = $breakpoint['width'];
+			$attr['height'] = $breakpoint['height'];
+			$attr['sizes']  = implode( ',', $breakpoint['sizes'] );
+
+			if ( 'mobile' === $device ) {
+				$attr['max'] = 499;
+			} elseif ( 'tablet' === $device ) {
+				$attr['min'] = 500;
+				$attr['max'] = 799;
+			} else {
+				$attr['min'] = 800;
+			}
+
+			$ad_html .= self::get_amp_ad( $attr, false, true );
```

Comment:
As per the function definition `get_amp_ad` function should have max 2 parameter but I am getting 3 here please check once.